projects/rocr-runtime/runtime/hsa-runtime/core/common/hsa_table_interface.cpp

```diff
@@ -1350,6 +1350,24 @@ hsa_status_t HSA_API hsa_amd_ais_file_read(hsa_amd_ais_file_handle_t handle, voi
                                            size_copied, status);
 }
 
+hsa_status_t HSA_API hsa_amd_counted_queue_acquire(
+    hsa_agent_t agent, hsa_queue_type_t type, hsa_amd_queue_priority_t priority,
+    void (*callback)(hsa_status_t status, hsa_queue_t* source, void* data), void* data,
+    uint64_t flags, hsa_queue_t** queue) {
+  return amdExtTable->hsa_amd_counted_queue_acquire_fn(agent, type, priority, callback, data, flags,
+                                                    queue);
+}
+
+hsa_status_t HSA_API hsa_amd_counted_queue_release(hsa_queue_t* queue) {
+  return amdExtTable->hsa_amd_counted_queue_release_fn(queue);
+}
+
+hsa_status_t HSA_API hsa_amd_counted_queue_get_info(hsa_queue_t* queue,
```

Comment:
We have an existing queue query API:
https://github.com/ROCm/rocm-systems/blob/develop/projects/rocr-runtime/runtime/hsa-runtime/inc/hsa_ext_amd.h#L3661

Please add the two new query types (HSA_QUEUE_INFO_USE_COUNT and HSA_QUEUE_INFO_HW_ID to the existing enum: hsa_queue_info_attribute_t